lib/matplotlib/tests/test_usetex.py

```diff
@@ -100,8 +100,22 @@ def test_usetex_packages(pkg):
                                   text.get_window_extent())
 
 
-def test_textcomp_full():
-    plt.rcParams["text.latex.preamble"] = r"\usepackage[full]{textcomp}"
+@pytest.mark.parametrize(
+    "latex_statement",
+    [r"\usepackage[full]{textcomp}", r"\usepackage{underscore}"],
+)
+def test_latex_pkg_already_loaded(latex_statement):
+    plt.rcParams["text.latex.preamble"] = latex_statement
```

Comment:
I'd just name this `preamble`:
```suggestion
@pytest.mark.parametrize(
    "preamble",
    [r"\usepackage[full]{textcomp}", r"\usepackage{underscore}"],
)
def test_latex_pkg_already_loaded(preamble):
    plt.rcParams["text.latex.preamble"] = preamble
```